src/functionalTest/java/uk/gov/hmcts/reform/cpo/befta/CaseIdGenerator.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.cpo.befta;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.util.Date;
+
+public class CaseIdGenerator {
```

Comment:
DO you also need to add your changes to the [CasePaymentOrdersTestAutomationAdapter.java](https://github.com/hmcts/cpo-case-payment-orders-api/blob/CPO-FTA/src/functionalTest/java/uk/gov/hmcts/reform/cpo/befta/CasePaymentOrdersTestAutomationAdapter.java) to make this useable to others?

Reply:
ahh good spot, had forgot to port across changes from that class onto this branch